### DISCUSSION TITLE
Resetting history

### DISCUSSION THREAD

**Discussion by AngelOnFira** [upptime/upptime#426] [Q&A]
I'm wondering if there is a proper way to reset history on a site. Like I was doing testing, and didn't want those downtime numbers to affect the production numbers. My solution was to do a `git filter-branch`, like such:

```
git filter-branch --index-filter "git rm -rf --cached --ignore-unmatch history/site.yml" HEAD
git push --force
```

Thanks!

Edit: Hmm it looks like the history is more tied together than I thought. I think I'll also need to remove the section from summary.json for each commit. I'll likely just reset all history then I think.

TLDR: Don't do this!

**Comment by voidrender** [comment]
One way that worked for me:

1. Permanently delete all of the GitHub issues associated with your testing.
2. Manually kick off the Setup CI action to run all the actions, which will recalculate all of the uptime stats.
